src/gtest/test_noteencryption.cpp

```diff
@@ -28,8 +31,8 @@ TEST(noteencryption, api)
         ASSERT_TRUE(b.get_epk() != c.get_epk());
     }
 
-    boost::array<unsigned char, 216> message;
-    for (unsigned char i = 0; i < 216; i++) {
+    boost::array<unsigned char, 201> message;
+    for (unsigned char i = 0; i < 201; i++) {
```

Comment:
Eh, to avoid silent integer overflow (which might not matter in this test anyway) I'd rather this be explicit for now so we can catch it later.